validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java

```diff
@@ -83,6 +84,7 @@ public class ValidatorConfig {
   private final boolean doppelgangerDetectionEnabled;
   private final boolean failoversSendSubnetSubscriptionsEnabled;
   private final boolean failoversPublishSignedDutiesEnabled;
+  private final boolean blockV3enabled;
```

Comment:
`blockV3Enabled` here and in next places would be more correct on case by my feelings

Reply:
Definitely! Fixed